index.html

```diff
@@ -5,38 +5,184 @@
+        <h3>"Welcome music lovers"</h3>
+        <h1 class="cocogoose">THE CONCERT OF THE CENTURY <br>THE MOST SPECTACULAR MEETING OF POP AND ROCK IN 2022</h1>
+        <p>A happy meeting with your favorite bands that will leave an unforgettable memory with a perfect sound that will reverberate throughout Colombia, only at the Arena Plaza in Bogota in October.</p>
+        <h2>2022.10.21(FRI) - 22(SAT)</h2>
+        <h4>Bogotá, Cota, Cundinamarca</h4>
+      </section>
+      <section class="program">
+        <h3>Main Program</h3>
+        <ul class="event-fase">
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <h5>Opening</h5>
+            <p>Inauguration of the event with the general director of the festival.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <h5>Musical History</h5>
+            <p>Here you will find the main milestones in the musical journey of our guest bands.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <h5>Store</h5>
+            <p>Only on this site can you buy souvenirs and musical products from your favorite band.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <h5>Main Event</h5>
+            <p>You can listen to all the bands Friday and Saturday from 4 pm and we will finish at 11 pm.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <h5>Closure</h5>
+            <p>Each of the bands in a special place will be autographing your favorite memories.</p>
+          </li>
+        </ul>
+        <a class="btn" href="#">Join to Concert of the Century</a>
+        <a class="btn-program" href="#main-page">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <section class="container__bands">;
+        <div id="featured">Featured Bands</div>
+        <section class="bands" id="bands" >
+        </section>
+      </section>
+      <footer>
+        <h3>Partner</h3>
+        <ul class="sponsors">
+          <li><a href="#"><img src="./images/mozilla-logo.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/google-logo.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/naver-logo.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/daumkakao-logo.png" alt=""></a></li>
+          <li><img src="./images/airbnb-logo.svg" alt=""></li>
+        </ul>
+      </footer>
+      <ul class="last-section">
+        <li>s
+          <img src="./images/logo-rock2.jpg" alt="">
+        </li>
+        <li>
+          <p>All content on this website, unless otherwise specified, is available under a Creative Commons Atrribution<br>Copyright © 2022. <br>All rights reserved.</p>
+        </li>
+      </ul>
+  </section>
+  <section class="about" id="about">
```

Comment:
- [ ] I appreciate the fact that your intention is to build a single-page website which is great 👍 .  I would like you to know that this project is a multi-page website and the project requirement specifies you to build two pages which in my opinion should be in their separate HTML files which means writing less logic and better organization just as the previous reviewer stated. Please add it 🤓 

![image](https://user-images.githubusercontent.com/63473584/181863013-0aec64ad-60e3-4c65-871b-50920ed36e91.png)

Reply:
Hello @vickymarz. Kindly show me where is the requirement that you say.  I do not get it.